### DISCUSSION TITLE
using prisma with other languages.

### DISCUSSION THREAD

**Discussion by dominiquegarmier** [prisma/prisma#13110] [Q&A] [answered]
Hello! Im building an app where my backend consists of multiple microservices (wirtten in potentially many different languages). So i am really just looking for a tool to build sql schemas in a more object oriented way. I was wondering if it made sense to use Prisma only for its schema definition language and schema migration tooling, while using raw sql queries (with different sql clients)?

**Comment by janpio** [comment, marked as answer]
Yep totally, if the schema parts give you some benefit, go ahead and use them without using Prisma Client.

**Comment by dominiquegarmier** [comment]
just found out about https://github.com/RobertCraigie/prisma-client-py and https://github.com/prisma/prisma-client-go aswell as obviously the official typescript and go clients, so that might be even better

**Reply by janpio** [reply]
Please take note of https://github.com/prisma/prisma-client-go/issues/707 - but yes, expect more of these one day in the future.

**Reply by dominiquegarmier** [reply]
hmm intresting. thanks for the headsup

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
